### DISCUSSION TITLE
Question: Accessing Notes Data Programmatically

### DISCUSSION THREAD

**Discussion by piyp791** [standardnotes/forum#3364] [General] [answered]
Hello, 
Is there a way to access the notes data programmatically? Looking to integrate bookmarked resources and notes across different apps and have a unified search. For other services, I could get an API to access my stored content. Couldn't find anything for standard notes.

**Comment by effieeee** [comment, marked as answer]
As far as I'm aware, there isn't an officially documented way to do this, but there have been some [3rd party developed extensions](https://github.com/jonhadfield/awesome-standard-notes#command-line) in the past which allowed you to access the notes on a CLI

**Reply by piyp791** [reply]
Those certainly look promising, Thanks!
